web/source/keyboards/kmwkeyboards.ts

```diff
@@ -267,10 +273,16 @@ namespace com.keyman.keyboards {
     getDetailedKeyboards() {
       var Lr = [], Ln, Lstub, Lrn;
 
-      for(Ln=0; Ln < this.keyboardStubs.length; Ln++)  // I1511 - array prototype extended
-      {
+      for(Ln=0; Ln < this.keyboardStubs.length; Ln++) { // I1511 - array prototype extended
         Lstub = this.keyboardStubs[Ln];
-        Lrn = this._GetKeyboardDetail(Lstub);  // I2078 - Full keyboard detail
+        let Lkbd;
+        for(let kbdIndex=0; kbdIndex < this.keyboards.length; kbdIndex++) {
+          if(this.keyboards[kbdIndex]['KI'] == Lstub['KI']) {
+            Lkbd = this.keyboards[kbdIndex];
+            break;
+          }
+        }
```

Comment:
```suggestion
        const Lkbd = this.keyboards.find(k => k['KI'] == Lstub['KI']);
```

Could also nove `Lstub` and `Lrn` declarations inside the loop which will help minification by reducing scope.

Reply:
Noting this for #6561; will make the change there, since that'll also swap some other blocks out for arrow functions.